### PR TITLE
Multi instance

### LINKED ISSUE #14: [feature] Allow for multiple Splunk instances.
Currently it appears you can only configure one Splunk instance in the configuration file.  It would be nice like SMTP if you could setup multiple instances and be able to chose from them.

### PR DESCRIPTION
**Pull request by satellite-no**
Bug:
- Fix for send_log entry point not matching file name.

Feature:
- Updated pack to support multiple Splunk instances. 

closes #14

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Change Log
 
+## 1.1.0
```
Comment (armab) [inline CHANGES.md:3]:
```suggestion
## 2.0.0
```
Per https://semver.org/ I recommend bumping the major version, considering it's a breaking change that changes the pack config schema completely.

**Review by armab — CHANGES_REQUESTED** [review]
Thanks for the contribution :+1: 

Just left one comment to address.

File: CHANGES.md (hunk 1 above)
Comment (satellite-no) [reply CHANGES.md:3]: Good call!

Comment (armab) [reply CHANGES.md:3]: Looks like Changelog version still needs a sync with the new pack.yaml version

**Review by armab — APPROVED** [review]
Thanks!

**Pull request merged**